`npm run build` fails on Windows (`mv` command is unknown)

Steps to reproduce: 
On Windows in cmd or in Terminal in VS Code run `npm run build` and wait.

Expected output: command successfully finishes.

Actual output:

```
>npm run build

> askql@1.2.1 prebuild C:\Users\Lukasz\Desktop\x\yonatan_askql
> ts-node scripts/preBuild.ts && tsc


> askql@1.2.1 build C:\Users\Lukasz\Desktop\x\yonatan_askql
> jest --config jest.build.config.js --no-cache && npm run playground:build

 PASS   build  src/askExpressMiddleware/askExpressMiddleware.spec.ts
 PASS   build  src/hello.ask.formatted.ask
 PASS   build  src/askscript/parser/askscript.grammar.pegjs
 PASS   build  src/hello.ask       

Test Suites: 4 passed, 4 total
Tests:       4 passed, 4 total
Snapshots:   0 total
Time:        3.557 s
Ran all test suites.

> askql@1.2.1 playground:build C:\Users\Lukasz\Desktop\x\yonatan_askql
> ts-node scripts/playgroundBuild.ts && npm run playground:build-assets && npm run playground:build-ts && ts-node scripts/playgroundPostBuild.ts


> askql@1.2.1 playground:build-assets C:\Users\Lukasz\Desktop\x\yonatan_askql
> npm run playground:build-assets:img


> askql@1.2.1 playground:build-assets:img C:\Users\Lukasz\Desktop\x\yonatan_askql
> imagemin src/playground/public/assets/img/* --out-dir=dist/playground/public/img

5 images minified

> askql@1.2.1 playground:build-ts C:\Users\Lukasz\Desktop\x\yonatan_askql
> tsc && mv dist/playground/public/assets/js dist/playground/public/js

'mv' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! askql@1.2.1 playground:build-ts: `tsc && mv dist/playground/public/assets/js dist/playground/public/js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the askql@1.2.1 playground:build-ts script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Lukasz\AppData\Roaming\npm-cache\_logs\2020-10-16T15_50_46_576Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! askql@1.2.1 playground:build: `ts-node scripts/playgroundBuild.ts && npm run playground:build-assets && npm run playground:build-ts && ts-node scripts/playgroundPostBuild.ts`npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the askql@1.2.1 playground:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Lukasz\AppData\Roaming\npm-cache\_logs\2020-10-16T15_50_46_764Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! askql@1.2.1 build: `jest --config jest.build.config.js --no-cache && npm run playground:build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the askql@1.2.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Lukasz\AppData\Roaming\npm-cache\_logs\2020-10-16T15_50_47_032Z-debug.log
```

The core error is here:
```
'mv' is not recognized as an internal or external command,
operable program or batch file.
```

Workaround:
Use Git Bash instead.

--- a/.github/workflows/build-on-windows.yml
+++ b/.github/workflows/build-on-windows.yml
@@ -1,0 +1,25 @@
+name: Build on Windows
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: windows-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out our repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Checkout and install dependencies
+        run: npm install
+
+      - name: build:test
+        run: npm run build:test

--- a/package.json
+++ b/package.json
@@ -27,7 +27,7 @@
     "playground:build": "ts-node scripts/playgroundBuild.ts && npm run playground:build-assets && npm run playground:build-ts && ts-node scripts/playgroundPostBuild.ts",
     "playground:build-assets": "npm run playground:build-assets:img",
     "playground:build-assets:img": "imagemin src/playground/public/assets/img/* --out-dir=dist/playground/public/img",
-    "playground:build-ts": "tsc && mv dist/playground/public/assets/js dist/playground/public/js",
+    "playground:build-ts": "tsc",
     "playground:start": "dotenv -e dist/.env node dist/playground/app.js",
     "build:all": "npm run build && npm run playground:build",
     "playground:server": "node dist/playground-backend/express/demoAskScriptServer.js"

--- a/scripts/playgroundPostBuild.ts
+++ b/scripts/playgroundPostBuild.ts
@@ -30,6 +30,11 @@ async function copyStaticAssets() {
       'src/playground/public/assets/css',
       'dist/playground/public/'
     );
+    shell.cp(
+      '-R',
+      'dist/playground/public/assets/js',
+      'dist/playground/public/js'
+    );
     shell.cp('-R', 'src/playground/views', 'dist/playground');
   } catch (e) {
     console.error(e);

--- a/scripts/postBuild.ts
+++ b/scripts/postBuild.ts
@@ -1,6 +1,5 @@
 import { promises as pfs } from 'fs';
 import { pick } from 'lodash';
-import fs from 'fs';
 
 import packageJson from '../package.json';
 

